### PR TITLE
Build: Port to 1.19.3 fabric

### PR DESCRIPTION
**Pull request by DJtheRedstoner**
Should be mostly functional, marked as draft until more extensive testing is done.

### CODE REVIEW

**Comment by DJtheRedstoner** [comment]
This is now ready for review.

**Review by cbyrneee — APPROVED** [review]
It all looks good to me from a code standpoint, I assume that you've gone through and checked that everything functions as expected. 

I don't see any issues with it other than `UKeyboard#allowRepeatEvents`. Can we not do:
```kt
//#if MC >= 11903
//$$ @Deprecated("Repeat keyboard events are always enabled in 1.19.3+")
//#endif
@JvmStatic
fun allowRepeatEvents(enabled: Boolean) {
```

Otherwise, someone won't be aware of the fact that they're always enabled on 1.19.3 and can't be disabled, probably causing someone a headache for 30 minutes until they check the UniversalCraft source.

File: src/main/kotlin/gg/essential/universal/UKeyboard.kt [hunk 1]
```diff
@@ -242,7 +242,7 @@ object UKeyboard {
     @JvmStatic
     fun allowRepeatEvents(enabled: Boolean) {
         //#if MC>=11903
-        //$$ // TODO: should this be no-op?
+        //$$ // Minecraft removed this function in 1.19.3, repeat events are now always enabled.
```
Comment (Sk1er) [inline src/main/kotlin/gg/essential/universal/UKeyboard.kt:245]: Should throw this in a KDoc on the function as well

**Review by Sk1er — CHANGES_REQUESTED** [review]

**Comment by DJtheRedstoner** [comment]
> I don't see any issues with it other than `UKeyboard#allowRepeatEvents`. Can we not do:
> 
> ```kotlin
> //#if MC >= 11903
> //$$ @Deprecated("Repeat keyboard events are always enabled in 1.19.3+")
> //#endif
> @JvmStatic
> fun allowRepeatEvents(enabled: Boolean) {
> ```
> 
> Otherwise, someone won't be aware of the fact that they're always enabled on 1.19.3 and can't be disabled, probably causing someone a headache for 30 minutes until they check the UniversalCraft source.

While this is an option, I'm not sure that deprecating the method really solve the issue. If it's deprecated on all versions, users will probably just ignore it or suppress the warning. If it's only deprecated on 1.19.3+, it is easy to miss the warning unless that version is your main version.

I've done what I think is a good compromise and added the note to the method kdoc, so it shows up as part of the method documentation and not just as part of the implementation.

Finally, before making this change I [searched GitHub](https://github.com/search?q=allowRepeatEvents&type=code) for users of this method, and as far as I can tell it's only used to opt-in to repeat events, not opt out. All Elementa guis [opt-in by default](https://github.com/EssentialGG/Elementa/blob/0799a87d607293c569635348e4e87b04eb6be974/src/main/kotlin/gg/essential/elementa/WindowScreen.kt#L20), and I [can't find](https://github.com/search?q=enableRepeatKeys&type=code) any notable examples on github that opt-out of this Elementa behavior.

**Review by cbyrneee — APPROVED** [review]

**Pull request merged**